external/storm-redis/src/main/java/org/apache/storm/redis/common/container/JedisCommandsContainer.java

```diff
@@ -12,24 +12,51 @@
 
 package org.apache.storm.redis.common.container;
 
+import redis.clients.jedis.GeoCoordinate;
+
 import java.io.Closeable;
-import redis.clients.jedis.JedisCommands;
+import java.util.List;
+
 
 /**
  * Interfaces for containers which stores instances implementing JedisCommands.
  */
-public interface JedisCommandsInstanceContainer extends Closeable {
-    /**
-     * Borrows instance from container.
-     * @return instance which implements JedisCommands
-     */
-    JedisCommands getInstance();
+public interface JedisCommandsContainer extends Closeable {
```

Comment:
Likely `JedisCommandsAdapter` is a better name here as we are providing a single unified interface ontop of both `JedisPool` and `JedisCluster` as Jedis removed the `JedisCommands` interface from covering both.  See https://github.com/xetorthio/jedis/pull/1932 for more details.